airbyte-workers/src/test/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowTest.java

```diff
@@ -1013,6 +1028,185 @@ void testGetStuckAfterRun(Consumer<ConnectionManagerWorkflow> signalSender, Thre
 
   }
 
+  @Nested
```

Comment:
thinking out loud: do you think it's worth breaking the reset tests out into a separate test class? this file is getting really big.

Reply:
TBD. I think it would be better to split the test class by type of child workflow. If we move everything in the appropriated nested class it should be fairly easy. I am also trying to factorize some elements in this review and in https://github.com/airbytehq/airbyte/pull/10864. I think that it could be better and make this file easier to read.

Reply:
I agree on splitting test class by type of child workflow. I don't think we need to do this in this PR.

Reply:
Spltting up the class will also help on run time, since Gradle parallelises at the class level.